lib/reek/ast/sexp_extensions/send.rb

```diff
@@ -0,0 +1,52 @@
+module Reek
+  module AST
+    module SexpExtensions
+      # Utility methods for :send nodes.
+      module SendNode
+        VISIBILITY_MODIFIERS = [:private, :public, :protected, :module_function]
+        ATTR_DEFN_METHODS = [:attr_writer, :attr_accessor]
+
+        def receiver; children.first; end
+        def method_name() children[1]; end
+        def args() children[2..-1] end
```

Comment:
Gaaah. I double-checked: this is not an example file to test whether the parser handless all three of these syntaxes. ;)